src/libsystemd/sd-network/sd-network.c

```diff
@@ -171,6 +171,18 @@ _public_ int sd_network_link_get_operational_state(int ifindex, char **state) {
         return network_link_get_string(ifindex, "OPER_STATE", state);
 }
 
+_public_ int sd_network_link_get_required_for_online(int ifindex) {
+        char* s;
+        int ret;
```

Comment:
I'd prefer it this was called `r`. That's what we use for plain int return codes.